src/NextGen/DonationForm/Blocks/DonationFormBlock/app/templates/elements/Paragraph.tsx

```diff
@@ -0,0 +1,9 @@
+import {ElementProps} from '@givewp/forms/propTypes';
+
+interface ParagraphProps extends ElementProps {
+    content: string;
+}
+
+export default function Paragraph({content}: ParagraphProps) {
+    return <p>{content}</p>;
```

Comment:
I wonder if it would simplify things to re-use the `html` element, since the only difference here is a wrapping tag. Not wanting to hold up this PR but something to consider/discuss.

Reply:
Yeah, I thought about that. Ultimately, I want different rules for what tags are supported. I'll probably use https://interweave.dev/